Sources/Testing/Running/Configuration.swift

```diff
@@ -258,6 +258,44 @@ public struct Configuration: Sendable {
 
   /// The test case filter to which test cases should be filtered when run.
   public var testCaseFilter: TestCaseFilter = { _, _ in true }
+
+  // MARK: - Expectation checking
+
+  /// Whether or not reflection of values in expressions checked by expectations
+  /// is enabled.
+  ///
+  /// When the value of this property is `false`, value reflection is disabled
+  /// and values will only be represented using `String(describing:)` instead of
+  /// using `Mirror` to recursively reflect their substructure.
+  public var isValueReflectionEnabled: Bool = true
+
+  /// The maximum number of elements that can included in a single child
+  /// collection when reflecting a value checked by an expectation.
+  ///
+  /// When ``Expression/Value/init(reflecting:)`` is reflecting a value and it
+  /// encounters a child value which is a collection, it consults the value of
+  /// this property and only includes the children of that collection up to this
+  /// maximum count. After this maximum is reached, all subsequent elements are
+  /// omitted and a single placeholder child is added indicating the number of
+  /// elements which have been truncated.
+  public var maximumValueReflectionCollectionCount: Int = 10
```

Comment:
```suggestion
  public var maximumValueReflectionCollectionCount = 10
```

Reply:
I like having the types of properties with `public` or greater access level state their type explicitly, to avoid the possibility of the inferred type changing without our knowledge and causing a API breakage. I see now that we have not done this with some trivial properties in `Configuration`, but now that I've moved most of these into a nested struct I've used this style within that new struct at least. I'd like to apply this coding style more broadly in a subsequent PR.

Reply:
We pretty much never do this for variables to which we assign values. We only specify types on variables that will be assigned values later. Consistency would be good (either way.)

Reply:
Already merged, but I did strip off the explicit types from these properties for now, for consistency!